sunpy/map/sources/tests/test_trace_source.py

```diff
@@ -0,0 +1,29 @@
+import os
+import glob
+
+from sunpy.map.sources.trace import TRACEMap
+from sunpy.map import Map
+import sunpy.data.test
+
+path = sunpy.data.test.rootdir
+fitspath = glob.glob(os.path.join(path, "tsi20010130_025823_a2.fits"))
+trace = Map(fitspath)
```

Comment:
this should be a [pytest fixture](https://docs.pytest.org/en/latest/fixture.html)